elastic/kibana
# [Uptime] Add Global Help support to uptime

## Summary

We want to add support for the Kibana help API for 7.0.

## Testing this PR
- Navigate to the Uptime app
- Click the global help icon in the upper right
- A link to Uptime's discuss forum should be displayed

![image](https://user-images.githubusercontent.com/18429259/54130146-3ad3aa00-43e6-11e9-925e-9ac2386798b2.png)


### Checklist

Use ~~strikethroughs~~ to remove checklist items you don't feel are applicable to this PR.


### For maintainers
